### PR TITLE
fix(deps): upgrade uuid to latest version

### PR DESCRIPTION
**Pull request by spalger**
## Summary

Upgrade UUID to avoid v3 warning:

```
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
```

## Details

Breaking changes:
 - Version 4-6 don't seem to exist based on the [changelog](https://github.com/uuidjs/uuid/blob/main/CHANGELOG.md)
 - Version 7 removed unused algorithms and deprecates the default export
 - Version 8 requires node 10 and removes the deprecated default export

One change was required, but there are only two uses of this lib so it wasn't a big deal.

### Checklist

none seem to be relevant

### CODE REVIEW

**Comment by nickofthyme** [comment]
Thanks @spalger I'll take a look at the CI errors.

**Comment by spalger** [comment]
Oh, oops, I needed to move to a remote dev instance and forgot to re-apply the changes to the existing uses of default imports

**Review by nickofthyme — APPROVED** [review]

File: jest.config.js [hunk 1]
```diff
@@ -27,4 +27,14 @@ module.exports = {
...
+  moduleNameMapper: {
+    // This forces Jest/jest-environment-jsdom to use a Node+CommonJS version of uuid, not a Browser+ESM one
+    // See https://github.com/uuidjs/uuid/pull/616
+    // See https://github.com/microsoft/accessibility-insights-web/pull/5421#issuecomment-1109168149
+    //
+    // WARNING: if your dependency tree has multiple paths leading to uuid, this will force all of them to resolve to
+    // whichever one happens to be hoisted to your root node_modules folder. This makes it much more dangerous
+    // to consume future uuid upgrades. Consider using a custom resolver instead of moduleNameMapper.
+    '^uuid$': require.resolve('uuid'),
+  },
```
Comment (nickofthyme) [inline jest.config.js:39]: Nice!

Comment (markov00) [reply jest.config.js:39]: that's what I've found working 😉

**Pull request merged**